### PR TITLE
fix #225, #227, #228

### LINKED ISSUE #225: Make it possible for Polymath to modify a ticker in the TR

### PR DESCRIPTION
**Pull request by satyamakgec**
- [x] remove swarmHash

- [x] modifyTickerDetails() added

- [x] unify the logging `LogNewSecurityToken`

### CODE REVIEW

File: contracts/SecurityTokenRegistry.sol [hunk 1]
```diff
@@ -35,10 +35,15 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, Util, Pausable, Regist
...
     // Emit when changePolyRegistrationFee is called
     event LogChangePolyRegistrationFee(uint256 _oldFee, uint256 _newFee);
-
     // Emit at the time of launching of new security token
-    event LogNewSecurityToken(string _ticker, address indexed _securityTokenAddress, address indexed _owner);
-    event LogAddCustomSecurityToken(string _name, string _symbol, address _securityToken, uint256 _addedAt);
+    event LogNewSecurityToken(
+        string _ticker,
+        string _name,
+        address indexed _securityTokenAddress,
```
Comment (pabloruiz55) [inline contracts/SecurityTokenRegistry.sol:42]: Is there any reason why address of the token should be indexed? @satyamakgec @VictorVicente

**Review by pabloruiz55 — APPROVED** [review]

File: contracts/SecurityTokenRegistry.sol (hunk 1 above)
Comment (satyamakgec) [reply contracts/SecurityTokenRegistry.sol:42]: I don't remember exactly but that was the requirement of the dApps team @bshevchenko

**Comment by pabloruiz55** [comment]
@satyamakgec I'm ok with merging it now

File: contracts/SecurityTokenRegistry.sol (hunk 1 above)
Comment (pabloruiz55) [reply contracts/SecurityTokenRegistry.sol:42]: We'll leave it.

**Pull request merged**